src/server/enterprise/server/api_server.go

```diff
@@ -330,3 +345,95 @@ func (a *apiServer) Deactivate(ctx context.Context, req *ec.DeactivateRequest) (
 
 	return &ec.DeactivateResponse{}, nil
 }
+
+func (a *apiServer) Pause(ctx context.Context, req *ec.PauseRequest) (resp *ec.PauseResponse, retErr error) {
+	if req.Namespace == "" {
+		req.Namespace = "default"
+	}
+	if err := a.setPauseState(ctx, req.Namespace, true); err != nil {
+		return nil, err
+	}
+	if err := scaleDownWorkers(ctx, a.env.GetKubeClient(), req.Namespace); err != nil {
```

Comment:
I think there may also be a race condition here, where we scale down the workers before pachds. I can imagine a situation where the pachds change the values of the worker RCs back to the spec before they are rolled over. 

I'm not sure what the best solution is here. We could scale down on startup, but that makes for a confusing control flow